src/lib/prov/pkcs11/p11.h

```diff
@@ -40,7 +47,7 @@
    #pragma pack(push, cryptoki, 1)
 #endif
 
-#include "pkcs11.h"
+#include <pkcs11.h>
```

Comment:
I can imagine the quote-based include might have been on purpose here. We do ship the upstream pkcs11 headers in the same include directory. There might be a case for preferring those over some system-provided ones. @randombit?

Reply:
I do not recall the reasoning on this one. I guess if we end up picking up a system provided header that's ok as long as it's recent enough.